schema/playbookRunResponse.message.yaml

```diff
@@ -65,7 +65,6 @@ properties:
         - end_line
 
 required:
-  - account
```

Comment:
the validator is no longer setting the account field, i.e. we can remove it not only from the required fields but we can remove the account field definition altogether